DO NOT MERGE: EDU-1180 Multi-region Namespace Epic (#2856)
temporalio/documentation: docs/production-deployment/cloud/introduction/sla.mdx

**What is Temporal Cloud's Service Level Agreement? SLA?**

> Two levels of availability are defined for Temporal Cloud: the [service availability](https://en.wikipedia.org/wiki/Reliability,_availability_and_serviceability) and the contractual [service level agreement](https://en.wikipedia.org/wiki/Service-level_agreement) (SLA).

Reviewer: active voice please.

Author: Acting on your suggestion.

> Two levels of availability are defined for Temporal Cloud: the [service availability](https://en.wikipedia.org/wiki/Reliability,_availability_and_serviceability) and the contractual [service level agreement](https://en.wikipedia.org/wiki/Service-level_agreement) (SLA).

Author: ```suggestion
Temporal Cloud provides two availability levels: the [service availability](https://en.wikipedia.org/wiki/Reliability,_availability_and_serviceability) and the contractual [service level agreement](https://en.wikipedia.org/wiki/Service-level_agreement) (SLA).
```

> Two levels of availability are defined for Temporal Cloud: the [service availability](https://en.wikipedia.org/wiki/Reliability,_availability_and_serviceability) and the contractual [service level agreement](https://en.wikipedia.org/wiki/Service-level_agreement) (SLA).

Author: ```suggestion
Temporal Cloud offers two availability levels: the [service availability](https://en.wikipedia.org/wiki/Reliability,_availability_and_serviceability) and the contractual [service level agreement](https://en.wikipedia.org/wiki/Service-level_agreement) (SLA).
```

- `PermissionDenied`
- `QueryFailed`
- `RetryReplication`
- `StickyWorkerUnavailable`
- `TaskAlreadyStarted`
- `Throttling (resources exhausted; triggers retry)`
- `WorkflowExecutionAlreadyStarted`
- `WorkflowNotReady`

> Our internal alerting system is based on a [service level objective](https://en.wikipedia.org/wiki/Service-level_objective) (SLO) for all errors, not just errors that count against the SLA.

Reviewer: capitalize Service Level Objective? Unsure.

Author: ![image](https://github.com/temporalio/documentation/assets/156466156/fbe56b18-382c-49b1-9f28-3ae07f86d2de)

Author: Thank you for the sanity check. I appreciate this kind of feedback.
